### ISSUE TITLE
the accuracy of (fold_id==6) in training is different from the accuracy of (fold_id==6) in reloading model parameters

### ISSUE THREAD

**Issue by Skyblue-tyut** [emadeldeen24/AttnSleep#14]
hi,
    accuracy in training (log and) is 0.8487006051975792, in reloading(reloading parameterts and testing) is 0.8447846208615165
    I replicated your work and noticed a strange phenomenon. I tested the accuracy of the validation set when the fold_id was 6, and the model parameters were the optimal model parameters saved during training. When I reload the model parameters, and use the code model.eval(). My predictions differ from those given by outs_6 and trgs_6. I wonder why this phenomenon occurs, I'll rule out the following reasons myself:
    1.The hardware equipment is the same, it's all my own laptop
    2.The data is loaded and used in the same way as it is during training
    3.model.eval(), This line of code has been used

<img width="423" alt="Snipaste_2023-11-08_22-57-11" src="https://github.com/emadeldeen24/AttnSleep/assets/51080399/0348c39a-3896-4e41-bfac-65dcba173bf6">

**Comment by emadeldeen24** [comment]
How about the other folds?

**Comment by emadeldeen24** [comment]
I'm closing the issue for now. Please reply later if the problem still exists.
